README.md

```diff
@@ -65,6 +65,26 @@ to build a cohesive PaaS. It is not a turn-key PaaS solution.
 See the [getting started](http://paasta.readthedocs.io/en/latest/installation/getting_started.html)
 documentation for how to deploy PaaSTA.
 
+## Debugging PaaSTA
```

Comment:
```suggestion
## Debugging PaaSTA (in VS Code)
```

Reply:
might also be nice to move this to the docs site (i.e., the readthedocs documentation), but this is probably fine for now until someone else wants to document how to debug paasta in their editor/ide of choice

Reply:
Thank you, it makes sense. I will remove everything under "Debugging PaaSTA" section and add it to internal wiki. Then under this section, I will mention that the document for debugging exists in internal wiki